test/coverage_test.js

```diff
@@ -6,5 +6,7 @@ import * as cornerstoneTools from '../src/index';
 describe('A test that pulls in all modules', function () {
   it('pulls in all modules', function () {
     expect(cornerstoneTools).to.exist;
+    expect(cornerstoneTools.drawing).to.exist;
```

Comment:
I don't think this stuff is necessary. That "test" is just a way to make Istanbul include all the files when it makes the coverage report.